tests/ci.rs

```diff
@@ -564,6 +564,7 @@ fn core() {
 }
 
 #[test]
+#[ignore]
```

Comment:
Is it not? I thought this test was pretty unreliable? Not actually sure if we want it disabled, but it fails pretty often when I try it.